pkg/transforms/policy.go

```diff
@@ -23,6 +24,12 @@ type PolicyResource struct {
 	node Node
 }
 
+type policyRelObj struct {
```

Comment:
I'm thinking we should make this more generic to re-use from other transforms. The basic concept is that we are creating a list of edges. We have everything except the UID, which we'll need to resolve later on buildEdges()

- Rename to something like `relatedObjectRef`
- Move to a different file, maybe `commonEdges.go` or `edgeHelper.go`

Reply:
Added type `relatedObject` and func `edgesFromRelatedObjects` to `common.go`.